backend/controllers/auth-controller.ts

```diff
@@ -164,12 +178,6 @@ export const isLoggedIn = asyncHandler(async (req, res) => {
       console.log(error);
     }
   }
-  const user = await User.findById(_id);
-  if (!user) {
```

Comment:
Why have we tottally removed this? please don't remove some functionality just because it is giving error XD

Reply:
not remove d but declared on above